### DISCUSSION TITLE
显示错误？

### DISCUSSION THREAD

**Discussion by shifcb** [share121/inter-knot#5375] [其他]
![Screenshot_20240924_180337_com share121 inter_knot](https://github.com/user-attachments/assets/0838a044-339a-4ac7-adfa-73c44c878e1f)
为什么我的显示未登录，有人了解吗？

**Comment by March-777th** [comment]
右滑找到**我的**，选择用户接口

**Reply by shifcb** [reply]
已经正常了，谢谢
